plutus-playground-lib/src/Playground/Interpreter/Util.hs

```diff
@@ -58,8 +58,8 @@ runTrace wallets actions =
                     emulatorLog = _emulatorLog newState
                     fundsDistribution =
                         Map.map
-                            (foldl' (+) 0 .
-                             fmap (Ada.fromValue . txOutValue) . view ownFunds) .
+                            (foldl' V.plus V.zero .
```

Comment:
nice

Reply:
Post https://github.com/input-output-hk/plutus/pull/797 you can just use `fold`